shell/layouts/default.vue

```diff
@@ -575,7 +588,11 @@ export default {
     <FixedBanner :header="true" />
     <AwsComplianceBanner v-if="managementReady" />
     <AzureWarning v-if="managementReady" />
-    <div v-if="managementReady" class="dashboard-content">
+    <div
+      v-if="managementReady"
+      class="dashboard-content"
+      :class="{[`pin-${ wmPin }`]: true}"
```

Comment:
nitpicking, but can we create a computed out of this? 🙏

Reply:
Sure, It's more clear with a `computed`.